content/en/providers/cloud-compute/openstack/useful_resources.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Useful Resources"
+weight: 60
+type: "docs"
+description: >
+  Collection of useful resources for OpenStack administrators
+---
+
+In this page we gather a list of useful resources for OpenStack administrators.
+
+## OpenStack IDLE VM Detector (osidle)
+
+Our partners from [GRyCAP UPV](https://grycap.upv.es/) have created a tool
+to analyze the usage of Virtual Machines running in OpenStack. For more
+information please visit:
+[https://github.com/grycap/osidle/](https://github.com/grycap/osidle/).
+
+## Share yours
+
+Would you like to share your favourite tool to make OpenStack management easier?
+Then [contact us](../../../../support/)!
```

Comment:
@sebastian-luna-valero can you check this link?

Reply:
fixed